### PR TITLE
Enable admin for caikit-tgis-client repo

### PR DESCRIPTION
**Pull request by Xaenalt**
Added caikit-tgis-client repo, need admin powers on it

### CODE REVIEW

**Review by accorvin — APPROVED** [review]

**Comment by Xaenalt** [comment]
@heyselbi PTAL

**Comment by Xaenalt** [comment]
This repo doesn't have that automation on it, can you hit approve on the code

File: config/opendatahub-io/org.yaml [hunk 1]
```diff
@@ -284,6 +284,7 @@ orgs:
           caikit: admin
           caikit-nlp: admin
           caikit-tgis-backend: admin
+          caikit-tgis-client: admin
```
Comment (dtrifiro) [inline config/opendatahub-io/org.yaml:287]: This should have been `caikit-nlp-client` https://github.com/opendatahub-io/caikit-nlp-client

**Pull request merged**